### PR TITLE
Refactor Gather DOW Data

### PR DESCRIPTION
**Pull request by juliusfitzhugh-ccpo**
### Refactor Gather `Dow` Data based on updated Dow document
- Update`AtatHelper` script include
- Create `Dow` script include
- Refactor `Gather DescriptionOfWork Data` action
- Update `Generate Description of Work` subflow

Ticket: AT-8381

### CODE REVIEW

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml [hunk 1]
```diff
@@ -0,0 +1,860 @@
...
+            packagingShippingOther: ctb(cc.packaging_shipping_other),
+            packagingShippingOtherExplanation:
+              cc.packaging_shipping_other_explanation.toString(),
+            contractorProvidedTransfer: ctb(cc.contractor_provided_transfer),
+            // 10c Supply Chain Risk Management 
+            // - no data, just text?
+
+            // 10d Training 
+            // - no data, just text?
+
```
Comment (zachclark-ccpo) [inline f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml:816]: Training: https://github.com/dod-ccpo/atat-snow/pull/229

Comment (juliusfitzhugh-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml:997]: The document appears to be just text (as if the data in #229 is included elsewhere).

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml [hunk 2]
```diff
@@ -0,0 +1,860 @@
...
+            // 10e
+            piiPresent: ctb(acqPackage.sensitive_information.pii_present),
+            systemOfRecordName: acqPackage.sensitive_information.system_of_record_name.toString(),
+            // 10f
+            travel: [],
```
[the remark this answers is not in the document]
Comment (juliusfitzhugh-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml]: Added in 9cdd2952277907ad74431ced48b7b2b2696e68c0.

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_9027655d97a7d5106fa8b4b3f153afba.xml [hunk 3]
```diff
@@ -131,12 +131,95 @@ AtatHelper.prototype = {
...
+      delete period.period_unit_count;
+      delete period.period_unit;
+      delete period.option_order;
+
+      return period;
+    });
+  },
+  /**
+   * Map periods ids for a given package to allow easier retreiveal for the
+   * different instances.
```
[the remark this answers is not in the document]
Comment (juliusfitzhugh-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_9027655d97a7d5106fa8b4b3f153afba.xml]: Update here 488104be4d7905a882b9e87fb5c45531812a7f1f

File: f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml (hunk 1 above)
Comment (juliusfitzhugh-ccpo) [reply f600233d1b154d507b782f84604bcb12/update/sys_script_include_3d46262b97a751106fa8b4b3f153af90.xml]: Confirmed with @luakim-ccpo that it's just hard coded text and no actual data. Check marked up template in ticket.

**Review by jasonburkert-ccpo — APPROVED** [review]

**Review by jeffsegal-ccpo — APPROVED** [review]

**Pull request merged**